webservices/args.py

```diff
@@ -139,15 +140,37 @@ def one_of(value, options):
     'party': Arg(str, multiple=True, description='Three-letter code for the party. For example: DEM=Democrat REP=Republican'),
 }
 
+filings = {
+    'committee_id': Arg(str, multiple=True, description=docs.COMMITTEE_ID),
+    'beginning_image_number': Arg(int, multiple=True, description=docs.BEGINNING_IMAGE_NUMBER),
+    'report_type': Arg(str, multiple=True, description='Report type'),
+    'report_year': Arg(int, multiple=True, description='Report year'),
+    'beginning_image_number': Arg(int, multiple=True, description=docs.BEGINNING_IMAGE_NUMBER),
+    'report_year': Arg(str, multiple=True, description='Year that the report applies to'),
+    'recipt_date': Arg(parse_date, multiple=True, description='The day the filing was received by the FEC'),
```

Comment:
Should be `receipt_date`. Also, in the schedules views, we're using `min_date` and `max_date` instead. Probably best to be consistent.